acceptance-tests/src/test/resources/features/site/datasets.feature

```diff
@@ -14,8 +14,11 @@ Scenario: Search individual data set by name
 
 Scenario: Download individual files from data set
```

Comment:
Scenario name could probably be updated to reflect the fact that it's not just about the files anymore.

Reply:
👍